src/DynamoCoreWpf/UI/DefaultBrandingResourceProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using Dynamo.Wpf.Interfaces;
+
+namespace Dynamo.Wpf.UI
+{
+    class DefaultBrandingResourceProvider : IBrandingResourceProvider
+    {
+        public ImageSource GetImageSource(ResourceName resourceName)
+        {
+            ImageSource image = null;
+            switch (resourceName)
+            {
+                case ResourceName.AboutBoxLogo:
+                    image = new BitmapImage(
+                    new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/AboutWindow/logo_about.png",
+                        UriKind.Absolute));
+                    break;
+            }
+            return image;
```

Comment:
Since this is a public API, it can be misused (imagine passing a string resource name in this method) and we need to highlight this problem earlier on. Can we add this right before it returns?

```csharp
if (image == null)
{
    throw new ArgumentInvalidException("resourceName", 
        String.Format("Resource name not handled: {0}", resourceName.ToString()));
}
```